Lombiq.Hosting.Tenants.FeaturesGuard/Handlers/FeaturesEventHandler.cs

```diff
@@ -189,4 +187,32 @@ public async Task DisableConditionallyEnabledFeaturesAsync(IFeatureInfo featureI
             await _shellFeaturesManager.DisableFeaturesAsync(currentlyEnabledConditionalFeatures);
         }
     }
+
+    private bool IsConditionallyEnabledFeaturesOptionsPopulated(out IDictionary<string, IEnumerable<string>> conditionalFeatures)
+    {
+        if (_conditionallyEnabledFeaturesOptions.Value.EnableFeatureIfOtherFeatureIsEnabled is not { } conditionallyEnabledFeatures)
+        {
+            conditionalFeatures = null;
+            return false;
+        }
+
+        conditionalFeatures = conditionallyEnabledFeatures;
+        return true;
+    }
+
+    private static IEnumerable<string> GetAllConditionFeatureIds(
```

Comment:
Here we're retrieving a distinct list of features that are to be conditionally enabled when `featureInfo` was enabled, right? If indeed, then apart from the name change also add this as docs on the method.

```suggestion
    private static IEnumerable<string> GetAllConditionalFeatureIds(
```

Reply:
No this is just a method to DRY the part where the condition features were collected in a new `var allConditionFeatureIds = new List<string>();`. So this is collecting the condition features not the conditional features. A conditinal feature would be `dictionary.Key` and not `dictionary.Value`

Reply:
I see, thanks.